examples/mongodbatlas_stream_privatelink_endpoint/confluent_serverless/main.tf

```diff
@@ -0,0 +1,59 @@
+resource "confluent_environment" "staging" {
+  display_name = "Staging"
```

Comment:
Should this be a variable?

Reply:
No strong opinion on this, but I think there's no need for this to be a variable. Users that already use the Confluent TF provider will have their environment already set up. I think the important part of this example is the new mongodb resource part

Reply:
IMO: `Staging` is a bit strange. I would prefer a placeholder (e.g., `my-env`) value to make it more obvious for the users to change it.